firebase-database/src/test/java/com/google/firebase/database/util/ComparatorTester.java

```diff
@@ -151,8 +150,8 @@ private final void doTestEquivalanceGroupOrdering() {
         testClassCast(reference);
         for (int otherIndex = 0; otherIndex < equalityGroups.size(); otherIndex++) {
           for (Object other : equalityGroups.get(otherIndex)) {
-            assertThat(Integer.signum(compare(reference, other)))
-                .named("compare(%s, %s)", reference, other)
```

Comment:
The `.named` method is deprecated. We're instead supposed to `assertWithMessage`.